src/dropin.js

```diff
@@ -133,6 +135,32 @@ Dropin.prototype._initialize = function (callback) {
   }.bind(this));
 };
 
+Dropin.prototype.updateConfig = function (prop, key, value) {
+  var methodsViewId = PaymentMethodsView.ID;
+
+  if (prop !== 'paypal' && prop !== 'paypalCredit') {
```

Comment:
I could see us making a whitelist in the constants just to make it abundantly clear what this is doing, but could be  overkill at this point.